### ISSUE TITLE
revealshapes: segmentation fault

### ISSUE THREAD

**Issue by jsbien** [rmast/djvulibre#2]
After uninstallink Inkscape I'm able to compile revealshapes. However call to it results immediately in segmentation fault.

**Comment by rmast** [comment]
Yes, I saw that. It needs a filename of a djvu file as parameter

**Comment by jsbien** [comment]
With a filename I get "Usage" (looks like old one for the database version). Which parameters should I use to get the cvs I'm looking for?
